### ISSUE TITLE
Program fail to run after upgrading system to Mac OS Mojave

### ISSUE THREAD

**Issue by MZLAKg2aGGFs8yi** [openconnect/openconnect-gui#250]
**Describe the bug**
Failed opening file ~/Library/Application Support/OpenConnect-GUI Team/OpenConnect-GUI/logs/OpenConnect-GUI.txt for writing: Operation timed out.

**To Reproduce**
1. Install openconnect-gui on mac os mojave
2. open openconnect-gui

**Expected behavior**
running

**Screenshots**
![image](https://user-images.githubusercontent.com/11944826/46124594-177bc080-c1f3-11e8-8237-c12ec62526e9.png)

**Desktop (please complete the following information):**
 - OS: Mac OS Mojave 10.14
 - Browser Chrome Version 69.0.3497.100 (Official Build) (64-bit)
 - Version 1.5.1

**Comment by horar** [comment]
I don't think it is related to Mojave upgrade. Anyway, this issue was already fixed in newest versions;

**Comment by naeemahmad1969** [comment]
Hi Horar, can you please tell me where to download the new version?

**Comment by horar** [comment]
the true is that macOS is not well supported in pre-build binary form. You can build it from source code. This will work.

If you want some pre-build packages then it's more complicated.
Probably outdated:
<pre>
    brew cask install openconnect-gui
</pre>

and most recent version (not HEAD!) version is mentioned [here](https://github.com/horar/homebrew-openconnect-gui). If it will work for you, let me know, thanks.
